services/network_config_manager/src/main/java/com/futurewei/alcor/netwconfigmanager/NetworkConfigManagerApplication.java

```diff
@@ -32,18 +32,23 @@
 @Import(TracerConfiguration.class)
 public class NetworkConfigManagerApplication {
 
+        /**
     @Autowired
     private NetworkConfigServer networkConfigServer;
 
     @PostConstruct
     public void instantiateGrpcServer(){
+        return;
+         * Return so that rest API endpoint can work
+         *
         try {
             networkConfigServer.start();
             networkConfigServer.blockUntilShutdown();
         } catch (Exception e) {
             e.printStackTrace();
         }
     }
+        */
```

Comment:
@pkommoju Why you comment out this part of code?

Reply:
Should we start grpcServer here?

Reply:
That comment should be removed, I will remove it.

Reply:
Let's just leave comment here. If we need it in the future, we can enable it then.

Reply:
This is a temporary change, it should have been removed but permanent change is coming from my other PR about NMM to NCM by REST. I will leave an exploratory comment disabling this section in there but leave this code in this PR as is.

Reply:
I think I will remove the comment here since the other PR will have a proper fix and SQL fields query doesn't strictly need NCM.